Formula/earthly.rb

```diff
@@ -17,14 +17,16 @@ class Earthly < Formula
 
   def install
     ldflags = "-X main.DefaultBuildkitdImage=earthly/buildkitd:v#{version} -X main.Version=v#{version} -X" \
-              " main.GitSha=7c9dd73df3cd27e070a6d7b22910f5853c0be9dd "
+              " main.GitSha=de7bfb10fcb6dc26b74aa2a76bc021773f8c9069 "
     tags = "dfrunmount dfrunsecurity dfsecrets dfssh dfrunnetwork"
     system "go", "build",
         "-tags", tags,
         "-ldflags", ldflags,
```

Comment:
Can you make a new pull request to replace the back ticks with `safe_popen_read`? This might fail and you'll just have an empty file.